api/iam.proto

```diff
@@ -121,11 +168,11 @@ message ChangePasswordResponse {
 }
 
 message GetPasswordResetRequest {
-	google.protobuf.StringValue reset_id = 1;
+	string reset_id = 1;
```

Comment:
Why string, we use google.protobuf.BytesValue before.

Reply:
If the parameter is defined in `path`, the type of the parameter should be `string`

```
option (google.api.http) = { get: "/v1/users/password/reset/{reset_id=*}" };
```